functions.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+func Getenv(key, fallback string) string {
+	value := os.Getenv(key)
+	if len(value) == 0 {
+		return fallback
+	}
+	return value
+}
+
+func GetDBConnector(dbFileName string) (*sql.DB, error) {
+	appPath, err := os.Getwd()
+	if err != nil {
+		fmt.Printf("ERROR SQL unable to find working directory\n")
+		return nil, err
+	}
+	dbFile := filepath.Join(appPath, dbFileName)
+	fmt.Printf("INFO SQL full path to DB file: %s", dbFile)
+	_, err = os.Stat(dbFile)
+	if err != nil {
+		fmt.Print("INFO SQL DB is missing. creating... ")
+		os.Create(dbFile)
+	}
+	dbCreator, errOpen := sql.Open("sqlite3", dbFile)
+	if errOpen != nil {
+		return nil, fmt.Errorf("unable to open DB \"%s\"", dbFileName)
```

Comment:
Совсем теряешь ошибки
```suggestion
		return nil, fmt.Errorf("unable to open DB \"%s\": %w", dbFileName, err)
```